src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
```

Comment:
you can use self-closing divs `<div />` when there is no children

Reply:
@choeqq hi, I've checked this information, and it looks like you cannot do it in pure .HTML file. I think it's only possible in react xhtml files :)